rust/src/lib.rs

```diff
@@ -201,6 +201,17 @@ mod ffi {
         fn state_checksum_changed(&self, chksum: &str, output_dir: &str) -> Result<bool>;
         fn update_state_checksum(&self, chksum: &str, output_dir: &str) -> Result<()>;
     }
+
+    // rpmutils.rs
+    extern "Rust" {
+        fn cache_branch_to_nevra(nevra: &str) -> String;
+    }
+
+    // rpmostree-rpm-util.h
+    unsafe extern "C++" {
```

Comment:
This is the new C++ binding; I think we'll do a section like this per header file or so?